recipes/gmx_MMPBSA/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "gmx_MMPBSA" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b166ec0a6a83d86e00865f54177dc99936882f4e2c478fedf35834eac0574f09
+
+build:
+  number: 0
+  skip: True # win
```

Comment:
The pre-processing selector should be `# [win]`.  I'm a bit surprised this worked but unless this is a new behavior I count that as a bug and would add the expected syntax to avoid breakages in the future.